graphql-ballerina/modules/parser/fragment_node.bal

```diff
@@ -73,4 +75,8 @@ public class FragmentNode {
     public isolated function getSelections() returns Selection[] {
         return self.selections;
     }
+
+    public isolated function isIncludeInlineFragment() returns boolean {
```

Comment:
Let's change this to `isInlineFragment` and the class variable to `inlineFragment`.